app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -4144,9 +4144,15 @@ private void useVideoSource(final boolean videoEnabled) {
 
         final int videoRenderIndex = getVideoRendererIndex();
```

Comment:
Please move this under the ``if (currentMetadata == null) { ... }`` check.
It will be useless when ``currentMeta`` is ``null``